frontend-react/src/__mocks__/DeliveriesMockServer.ts

```diff
@@ -2,6 +2,21 @@ import { rest } from "msw";
 import { setupServer } from "msw/node";
 
 import { RSDelivery } from "../network/api/History/Reports";
+import { RSFacility } from "../config/endpoints/deliveries";
+import config from "../config";
+
+const { RS_API_URL } = config;
+
+const makeFacilityFixture = (
```

Comment:
Toying around with ways we can generate unique fixtures and where to store them.